pennylane_qiskit/basic_aer.py

```diff
@@ -58,10 +58,56 @@ def __init__(self, wires, shots=1024, backend="qasm_simulator", **kwargs):
 
         if Version(qiskit.__version__) > max_ver:
             raise RuntimeError(
-                f"The devices in the PennyLane Qiskit plugin are currently only compatible "
-                f"with versions of Qiskit below 0.46. You have version {qiskit.__version__} "
-                f"installed. Please downgrade Qiskit to use the devices. The devices will be "
-                f"updated in the coming weeks to be compatible with Qiskit 1.0!"
+                f"Qiskit has discontinued the BasicAer device, so it can only be used in"
+                f"versions of Qiskit below 0.46. You have version {qiskit.__version__} "
+                f"installed. For a Python simulator, use the 'qiskit.basicsim' device "
+                f"instead. Alternatively, you can downgrade Qiskit to use the "
+                f"'qiskit.basicaer' device."
             )
 
         super().__init__(wires, provider=qiskit.BasicAer, backend=backend, shots=shots, **kwargs)
+
+
+class BasicSimulatorDevice(QiskitDevice):
+    """A PennyLane device for the native Python Qiskit simulator BasicSimulator.
+
+    Please see the `Qiskit documentations <https://qiskit.org/documentation/>`_
+    further information on the backend options and transpile options.
```

Comment:
```suggestion
    Please see the `device documentation <https://qiskit.org/documentation/>`_
    for further information on the backend options.
```
Do we have a better URL?

Reply:
Actually this PR is WIP so I'll stop commenting for now and look back later.

Reply:
> Do we have a better URL?

I added a link to the BasicProvider instead, it has more specific documentation about options for this device